spec/unit/provider/service/systemd_spec.rb

```diff
@@ -171,6 +171,13 @@
     end
   end
 
+  it "should be the default provider on raspbian12" do
+    allow(Facter).to receive(:value).with(:osfamily).and_return(:debian)
+    allow(Facter).to receive(:value).with(:operatingsystem).and_return(:raspbian)
+    allow(Facter).to receive(:value).with(:operatingsystemmajrelease).and_return("12")
```

Comment:
Puppet 8 (main) uses structured facts so these stubs need to be updated. Something like:

```diff
-    allow(Facter).to receive(:value).with(:osfamily).and_return(:debian)
-    allow(Facter).to receive(:value).with(:operatingsystem).and_return(:raspbian)
-    allow(Facter).to receive(:value).with(:operatingsystemmajrelease).and_return("12")
+    allow(Facter).to receive(:value).with(:os.family).and_return(:debian)
+    allow(Facter).to receive(:value).with(:os.name).and_return(:raspbian)
+    allow(Facter).to receive(:value).with(:os.release.major).and_return("12")
```

Reply:
@jaevans will you take care or have you given up (for whatever reason)?